templates/svc.yaml

```diff
@@ -10,6 +10,12 @@ metadata:
     release: {{ .Release.Name }}
 spec:
   type: {{ .Values.service.type }}
+  {{- if .Values.service.loadBalancerIP }}
+  loadBalancerIP: {{ .Values.service.loadBalancerIP }}
+  {{- end }}
+  {{- if (or (eq .Values.service.type "LoadBalancer") (eq .Values.service.type "NodePort")) }}
+  externalTrafficPolicy: {{ .Values.service.externalTrafficPolicy | quote }}
```

Comment:
This is GCE specific flag. 

External services are provider dependant hence we need to be careful. For instance, AWS allows to define whitelisted ips but it will not validate the yaml if that section is present trying to create an external service somewhere else. 

This is just a heads up that we need to test this properly in different environments as it would crash the whole installation.

Reply:
Thanks! I'll remove it then.